src/odemis/gui/comp/viewport.py

```diff
@@ -961,7 +965,7 @@ def _on_stream_update(self, _):
         ss = self._view.getStreams()
         if len(ss) > 0:
             # Any stream not static?
-            show = any(not isinstance(s, StaticStream) for s in ss)
+            show = any(not isinstance(s, (StaticStream, DataProjection)) for s in ss)
```

Comment:
I had to modify getStreams to also return projections - otherwise, nothing was displaying. 

Here, without this change, the pause button is shown on view ports with projections.